src/components/WalletView/WalletView.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import {
+  Badge,
+  Box,
+  Flex,
+  Heading,
+  Text,
+  useColorModeValue
+} from '@chakra-ui/react';
+
+import { i18n } from '@/i18n';
+
+export const WalletView = (props) => {
+  const bgMain = useColorModeValue('orange.50', 'teal.800');
+  const textColor = useColorModeValue('teal.900', 'orange.100');
+  const bgBadge = useColorModeValue('orange.100', 'teal.700');
+
+  const totalBalanceView = new Intl.NumberFormat('de-DE', {
```

Comment:
Why do you use 'de-DE' and not 'en-US' here?
'en-US' is more suitable for $

Reply:
We have to "Display two digits after the decimal point (e.g. 0.000,00) with code  " and if I change to 'en-US' it would be another format

Reply:
So if it's PO's requirement - I have no questions here